solr/CHANGES.txt

```diff
@@ -183,6 +183,10 @@ Upgrade Notes
 
 * SOLR-14065: VelocityResponseWriter has been deprecated and may be removed in a future version.
 
+* SOLR-14072: The "Blob Store" API and "runtimeLib" plugin mechanism that uses it is now considered deprecated.  The
```

Comment:
Should we even deprecare the blob store API itself? I think Metrics system uses it. @sigram any thoughts/ideas?

Reply:
I'm +1 to deprecating it, though. Please go ahead with merge.